src/06-extensions.md

```diff
@@ -0,0 +1,47 @@
+# Contributing to and extending the BIDS specification
+
+This page lists some ways that you can get involved with the BIDS community.
+
+## The BIDS Contributor guide
```

Comment:
I think here it would also make sense to point to and distinguish from the BIDS starter kit, since people can contribute to BIDS without proposing an extension. WDYT ?